app/views/lab_tests/_date_filter.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="mb-6 flex justify-between items-center">
+    <%= form_with(url: lab_tests_path, method: :get, class: "inline-flex items-center") do |f| %>
+    <div class="inline-flex items-center">
+        <span class="mr-3 text-sm font-medium"><%= t('.lab_test_date_filter.from') %></span>
+        <%= f.date_field :start_date, 
+          value: params[:start_date] || Rails.application.config.x.lab_test_date_range[:default_start_date].call,
+          min: Rails.application.config.x.lab_test_date_range[:min_date],
+          max: Rails.application.config.x.lab_test_date_range[:max_date],
+          onchange: "this.nextElementSibling.nextElementSibling.min=this.value",
+          class: "rounded-md border border-gray-300 py-2 px-3 mr-6",
+          placeholder: "mm/dd/yyyy" %>
```

Comment:
`placeholder: "mm/dd/yyyy"` this could be extracted to config as well to easily change for non-American users.
Not critical for this PR tho.